data/release-notes/enterprise-server/3-2/0-rc1.yml

```
@@ -61,7 +61,7 @@ sections:
         - |
           Git Credential Manager (GCM) Core versions 2.0.452 and later now provide security-hardened credential storage and multi-factor authentication support for {% data variables.product.product_name %}.
```

I suggest we remove the reference here also.
```suggestion
          Git Credential Manager (GCM) versions 2.0.452 and later now provide security-hardened credential storage and multi-factor authentication support for {% data variables.product.product_name %}.
```